src/components/ec-loading/ec-loading.vue

```diff
@@ -0,0 +1,76 @@
+<template>
+  <div class="ec-loading__container">
```

Comment:
We should use elements inside the block, so the __container should be inside ec-loading. Give it better name or make this one `ec-loading` and `ec-loading__content` the nested one.